ofl/akatab/METADATA.pb

```diff
@@ -0,0 +1,67 @@
+name: "Akatab"
+designer: "SIL International"
+license: "OFL"
+category: "SANS_SERIF"
+date_added: "2023-02-15"
+fonts {
+  name: "Akatab"
+  style: "normal"
+  weight: 400
+  filename: "Akatab-Regular.ttf"
+  post_script_name: "Akatab-Regular"
+  full_name: "Akatab"
+  copyright: "Copyright (c) 2010-2023, SIL International (https://scripts.sil.org), with Reserved Font Name \"Akatab\", copyright 2010-2019 Adobe (http://www.adobe.com/), with Reserved Font Name \"Source\" (Latin glyphs)."
+}
+fonts {
+  name: "Akatab"
+  style: "normal"
+  weight: 500
+  filename: "Akatab-Medium.ttf"
+  post_script_name: "Akatab-Medium"
+  full_name: "Akatab Medium"
+  copyright: "Copyright (c) 2010-2023, SIL International (https://scripts.sil.org), with Reserved Font Name \"Akatab\", copyright 2010-2019 Adobe (http://www.adobe.com/), with Reserved Font Name \"Source\" (Latin glyphs)."
+}
+fonts {
+  name: "Akatab"
+  style: "normal"
+  weight: 600
+  filename: "Akatab-SemiBold.ttf"
+  post_script_name: "Akatab-SemiBold"
+  full_name: "Akatab SemiBold"
+  copyright: "Copyright (c) 2010-2023, SIL International (https://scripts.sil.org), with Reserved Font Name \"Akatab\", copyright 2010-2019 Adobe (http://www.adobe.com/), with Reserved Font Name \"Source\" (Latin glyphs)."
+}
+fonts {
+  name: "Akatab"
+  style: "normal"
+  weight: 700
+  filename: "Akatab-Bold.ttf"
+  post_script_name: "Akatab-Bold"
+  full_name: "Akatab Bold"
+  copyright: "Copyright (c) 2010-2023, SIL International (https://scripts.sil.org), with Reserved Font Name \"Akatab\", copyright 2010-2019 Adobe (http://www.adobe.com/), with Reserved Font Name \"Source\" (Latin glyphs)."
+}
+fonts {
+  name: "Akatab"
+  style: "normal"
+  weight: 800
+  filename: "Akatab-ExtraBold.ttf"
+  post_script_name: "Akatab-ExtraBold"
+  full_name: "Akatab ExtraBold"
+  copyright: "Copyright (c) 2010-2023, SIL International (https://scripts.sil.org), with Reserved Font Name \"Akatab\", copyright 2010-2019 Adobe (http://www.adobe.com/), with Reserved Font Name \"Source\" (Latin glyphs)."
+}
+fonts {
+  name: "Akatab"
+  style: "normal"
+  weight: 900
+  filename: "Akatab-Black.ttf"
+  post_script_name: "Akatab-Black"
+  full_name: "Akatab Black"
+  copyright: "Copyright (c) 2010-2023, SIL International (https://scripts.sil.org), with Reserved Font Name \"Akatab\", copyright 2010-2019 Adobe (http://www.adobe.com/), with Reserved Font Name \"Source\" (Latin glyphs)."
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "tifinagh"
+source {
+  repository_url: "https://github.com/silnrsi/font-akatab"
+  archive_url: "https://github.com/silnrsi/font-akatab/releases/download/v3.000/Akatab-3.000.zip"
+}
```

Comment:
@emmamarichal primary script field is missing. I'll explain to you wha is it for and how to add it.